### ISSUE TITLE
Start Script not working

### ISSUE THREAD

**Issue by derekbtw** [viljamis/vue-design-system#42]
```
C:\Users\btw\Dev\vue\vue-design-system>npm start

> vue-design-system@1.3.1 start C:\Users\btw\Dev\vue\vue-design-system
> bash start.sh

npm ERR! code ELIFECYCLE
npm ERR! errno 3221225775
npm ERR! vue-design-system@1.3.1 start: `bash start.sh`
npm ERR! Exit status 3221225775
npm ERR!
npm ERR! Failed at the vue-design-system@1.3.1 start script.
```

**2018-03-19T22_54_56_380Z-debug.log**

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\btw\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@5.7.1
3 info using node@v8.9.3
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle vue-design-system@1.3.1~prestart: vue-design-system@1.3.1
6 info lifecycle vue-design-system@1.3.1~start: vue-design-system@1.3.1
7 verbose lifecycle vue-design-system@1.3.1~start: unsafe-perm in lifecycle true
8 verbose lifecycle vue-design-system@1.3.1~start: PATH: C:\Users\btw\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Users\btw\Dev\vue\vue-design-system\node_modules\.bin;C:\Python27\;C:\Python27\Scripts;C:\Users\btw\AppData\Local\Programs\Python\Python36-32\;C:\Program Files\Ruby24-x64\bin;C:\Program Files (x86)\Razer Chroma SDK\bin;C:\Program Files\Razer Chroma SDK\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Users\btw\AppData\Local\Microsoft\WindowsApps;C:\cygwin64\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Users\btw\apps;C:\ProgramData\chocolatey\bin;C:\Program Files (x86)\Yarn\bin;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\WINDOWS\System32\OpenSSH\;C:\WINDOWS\System32\LibreSSL\;C:\Users\btw\AppData\Local\Programs\Python\Python36-32\Scripts\;C:\Users\btw\AppData\Local\atom\bin;C:\Users\btw\AppData\Local\GitHubDesktop\bin;C:\Users\btw\AppData\Local\Yarn\bin;C:\Users\btw\AppData\Roaming\npm
9 verbose lifecycle vue-design-system@1.3.1~start: CWD: C:\Users\btw\Dev\vue\vue-design-system
10 silly lifecycle vue-design-system@1.3.1~start: Args: [ '/d /s /c', 'bash start.sh' ]
11 silly lifecycle vue-design-system@1.3.1~start: Returned: code: 3221225775  signal: null
12 info lifecycle vue-design-system@1.3.1~start: Failed to exec start script
13 verbose stack Error: vue-design-system@1.3.1 start: `bash start.sh`
13 verbose stack Exit status 3221225775
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\btw\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:285:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\btw\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid vue-design-system@1.3.1
15 verbose cwd C:\Users\btw\Dev\vue\vue-design-system
16 verbose Windows_NT 10.0.17115
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\btw\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
18 verbose node v8.9.3
19 verbose npm  v5.7.1
20 error code ELIFECYCLE
21 error errno 3221225775
22 error vue-design-system@1.3.1 start: `bash start.sh`
22 error Exit status 3221225775
23 error Failed at the vue-design-system@1.3.1 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 3221225775, true ]

```

**Comment by viljamis** [comment]
@derekbtw Let me look into this. I haven’t had time yet to test everything on Windows, but will make this a priority right now.

**Comment by viljamis** [comment]
@derekbtw Created a new roadmap issue for this: https://github.com/viljamis/vue-design-system/issues/43

**Comment by viljamis** [comment]
@derekbtw Just tested and added support for Windows. There were a few broken things: YAML tokens and their watch task. Those are fixed now. 

I believe the issue you’re seeing here is caused by Windows Command Prompt (or something else) not recognizing the shell scripts (.sh) used to start Vue Design System. To get around this on Windows you should be able to start Vue Design System for example in `Git Bash` that can read those. 

Found at least this post that hopefully helps to install Git Bash on Windows _(it should be fairly simple, took me less than half a minute):_ http://www.techoism.com/how-to-install-git-bash-on-windows/ Do let me know if you will try this solution and if you still run into other issues.

I might consider getting rid of the shell script requirement at some point, but currently it’s a more robust and simpler solution than using for example npm-run-all or concurrently to run multiple npm-scripts in parallel. I were using both of those previously, but they caused a lot of issues. I can revisit this and do more investigation at some point though.

**Comment by viljamis** [comment]
@derekbtw I updated the getting started guide to reflect this as well: https://github.com/viljamis/vue-design-system/wiki/Getting-Started#getting-started

**Comment by viljamis** [comment]
This will be simplified in 2.0.0 release (no need for shell scripts anymore)

**Comment by viljamis** [comment]
Released in [Vue Design System 2.0.0](https://github.com/viljamis/vue-design-system/releases/tag/2.0.0). Shouldn’t require anything special on Windows anymore. Same instructions as for macOS work now.

**Comment by derekbtw** [comment]
Sweet! I’ll check it out tonight when I get home. Thanks 👍
